app/models/healthcheck/storage_check.rb

```diff
@@ -5,13 +5,28 @@ def name
     end
 
     def status
-      healthcheck_key = 'healthcheck.txt'
-
-      unless SelfService.service(:storage_client).exist?(healthcheck_key)
-        SelfService.service(:storage_client).upload(healthcheck_key, '')
+      Rails.configuration.hub_environments.values.each do |bucket|
```

Comment:
why not
```        next unless healthcheck_file_exists?(bucket)

        SelfService.service(:storage_client).put_object(
          bucket: bucket,
          key: FILES::HEALTHCHECK,
          body: ''
        )
```

Reply:
In this case we want the healthcheck to actually try to find a specific file in a bucket and if it doesn't exist to try to upload a file there. So `bucket` at this point will be the name that the AWS SDK needs in order to connect correctly.